components/SidebarNav/SidebarMenuGroup.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { Collapse } from '@mui/material';
+import { useRouter } from 'next/router';
+import { HeroIcon } from '../HeroIcon';
+import { SidebarMenuLink, SidebarMenuLinkProps } from './SidebarMenuLink';
+import * as styles from './SidebarNav.styles';
+
+export interface SidebarMenuGroupProps {
+  href?: string;
+  label: string;
+  menuLinks?: SidebarMenuLinkProps[];
+  parentPath?: string;
+  className?: string;
+}
+
+export const SidebarMenuGroup = ({
+  href,
+  label,
+  menuLinks,
+  parentPath,
+  className,
+}: SidebarMenuGroupProps) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const toggleExpanded = () => setIsExpanded(!isExpanded);
+
+  const router = useRouter();
+
+  // Expand the child menu if the current page is a child of the parentPath
+  useEffect(() => {
+    if (parentPath && router.asPath.includes(parentPath)) {
+      setIsExpanded(true);
+    }
+  }, [parentPath, router.asPath]);
```

Comment:
This makes sure that if you go directly to a 2nd level page, e.g., 
https://deploy-preview-129--dev-decanter-v7.netlify.app/for-developers/strategies-and-conventions

that section of the nav would be expanded
![Screen Shot 2023-01-03 at 6 01 29 PM](https://user-images.githubusercontent.com/42749717/210471075-ee7adafe-88e9-465e-b25d-e32442d3738d.png)